android/app/build.gradle

```diff
@@ -105,6 +105,16 @@ android {
             abiFilters "armeabi-v7a", "x86"
         }
     }
+    signingConfigs {
+        release {
+            if (project.hasProperty('MYAPP_RELEASE_STORE_FILE')) {
+                storeFile file(MYAPP_RELEASE_STORE_FILE)
+                storePassword MYAPP_RELEASE_STORE_PASSWORD
```

Comment:
I actually used [this](https://pilloxa.gitlab.io/posts/safer-passwords-in-gradle/) approach when setting up a password for release. Not sure how we can have that flow integrated into this PR or even if we should as it's  quite user specific :thinking: